test/test_pot.py

```diff
@@ -0,0 +1,29 @@
+from testing_base import *
+
+# Create a deck to use
+deck = Deck()
+
+# Create pot
+pot = Pot(10)
+
+jesse = Player("Jesse", 2)
+
```

Comment:
According to Prof Lee, its better practice to have these initializing things be in each test function. This might be a good thing to consider.